app/Resources/views/article/article.html.twig

```diff
@@ -77,6 +77,10 @@
 
     </section>
 
+    <section class="macron__banner macron__banner--row l__wrapper--narrow">
+        {{ include('banner/macron.html.twig') }}
+    </section>
+    {#
```

Comment:
Instead of commenting, please remove code. We will have the git history if needed.